packages/firebase_remote_config/lib/src/remote_config_fetch_throttled_exception.dart

```diff
@@ -0,0 +1,18 @@
+part of firebase_remote_config;
+
+class FetchThrottledException implements Exception {
+  DateTime _throttleEnd;
+
+  FetchThrottledException._({int endTimeInMills = 43200}) {
```

Comment:
I am not sure why there's a default value of `43200` here. It seems like we are already specifying a value in the one place where this exception is constructed, so you can just remove this.

Reply:
Good point, Done